core/conversion/conversionctx/ConversionCtx.cpp

```diff
@@ -45,6 +48,25 @@ ConversionCtx::ConversionCtx(BuilderSettings build_settings)
           "[TRTorch Conversion Context] - ",
           util::logging::get_logger().get_reportable_severity(),
           util::logging::get_logger().get_is_colored_output_on()) {
+    // Get list of all available plugin creators
+
+    initLibNvInferPlugins(&logger, "");
```

Comment:
I suspect that this will need to occur in a more global manner since it will need to cover cases were we only use the runtime and the conversion context is therefore never created. We also probably should use the global logger in that case as well instead of the conversion logger.

Reply:
Ideally this would be located somewhere in the runtime module and run on library load since we would also want this for libtrtorchrt.so as well.

Reply:
I am not sure where is the best place to initialize the plugin. Can you suggest a specific place?